lib/client.ts

```diff
@@ -96,32 +103,31 @@ export default class Client {
   }
 
   public leaveGroup(groupId: string): Promise<any> {
-    return this.post(URL.leaveGroup(groupId));
+    return this.post(URL.leaveGroup(groupId)).then(this.checkJSON);
```

Comment:
We needn't care about response here (anyway the response will be empty, thus return type is `any`), so just remove the check.